diff --git a/node_modules/kysely-d1/dist/index.js b/node_modules/kysely-d1/dist/index.js
index 8fecbb3..1bd50f0 100644
--- a/node_modules/kysely-d1/dist/index.js
+++ b/node_modules/kysely-d1/dist/index.js
@@ -85,9 +85,9 @@ class D1Connection {
         if (results.error) {
             throw new Error(results.error);
         }
-        const numAffectedRows = results.changes > 0 ? BigInt(results.changes) : undefined;
+        const numAffectedRows = results.meta.changes > 0 ? BigInt(results.meta.changes) : undefined;
         return {
-            insertId: results.lastRowId === undefined || results.lastRowId === null ? undefined : BigInt(results.lastRowId),
+            insertId: results.meta.last_row_id === undefined || results.meta.last_row_id === null ? undefined : BigInt(results.meta.last_row_id),
             rows: results?.results || [],
             numAffectedRows,
             // @ts-ignore deprecated in kysely >= 0.23, keep for backward compatibility.
